src/main/java/organice/model/person/OrganExpiryDate.java

```diff
@@ -30,7 +30,7 @@ public OrganExpiryDate(String expiryDate) {
         try {
             DateFormat dateFormat = new SimpleDateFormat("dd MMM yyyy");
             checkArgument(isValidExpiryDate(expiryDate), MESSAGE_CONSTRAINTS);
-            value = dateFormat.format(DATE_FORMAT.parse(expiryDate)); //make the data value uniform
+            value = dateFormat.format(DATE_FORMAT.parse(expiryDate)).replace(" ", "-"); //make the data value uniform
```

Comment:
I don't quite get this line, what is the purpose of replacing the spaces with the dashes?

Reply:
Oh cause, when it is saved in the json, if the format is `22 Jan 2019` it will throw error since its not `DD-MMM-YYYY` format, so need to save as `DD-MMM-YYYY` format as well

Reply:
Noted